requirements.txt

```diff
@@ -1,2 +1,6 @@
 wcwidth>=0.1.4
 six>=1.9.0
+# support python2.6 by using backport of 'orderedict'
+ordereddict==1.1; python_version < "2.7"
+# Windows requires jinxed
+jinxed>=0.5.4; platform_system == "Windows"
```

Comment:
Well I hope you will do nicely and not break `jinxed` any day in the future :)

I don't typically care for these `>=`, anything could happen in the future, nefarious, even, but I'm a bit FOSS tired and feeling more trusting these days.